compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeAttention.cpp

```diff
@@ -161,6 +161,12 @@ static Value computeQKTranspose(Value query, Value key, Value output,
   return matmulOp.getResult(0);
 }
 
+static Value maskQKTranspose(Value qkResult, Value mask, Location loc, OpBuilder &builder, SmallVectorImpl<Operation *> &ops) {
```

Comment:
Do we need these changes in this file, since we are dependent on the online attention decomposition anyways

Reply:
Nope seems to work fine without it. Confirming it can be removed?